mod-orders/examples/bindPiecesCollection.sample

```diff
@@ -5,7 +5,8 @@
     "materialTypeId": "a058cb27-a77d-4cb3-b76b-2adc931fb0c9",
     "callNumber": "BF2050 .M335 2000",
     "permanentLoanTypeId": "41673a86-ebab-4a5a-8b1b-76110a6f86f4",
-    "permanentLocationId": "02cc63e9-dda0-4b49-bbc6-6253edffcc67",
+    "locationId": "02cc63e9-dda0-4b49-bbc6-6253edffcc67",
+    "holdingId": "0804ddec-6545-404a-b54d-a693f505681d",
```

Comment:
only locationId or holdingId can be presented, but not both at the same time

Reply:
Is there any way to have two sample files? Or only one is supported to be displayed?

Reply:
you can remain for example holdingId and it is enough